Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Types/FarePolicy.hs

```diff
@@ -44,12 +55,74 @@ data FarePolicyD (s :: DTC.UsageSafety) = FarePolicy
   }
   deriving (Generic, Show)
 
+jsonToDriverExtraFeeBounds :: String -> String -> Maybe (NonEmpty DriverExtraFeeBounds)
+jsonToDriverExtraFeeBounds config key' =
+  let res' =
+        config
+          ^@.. _Value
```

Comment:
```
_Value
            . _Object
            . reindexed
              (dropPrefixFromConfig "farePolicyDriverExtraFeeBounds:")
              ( itraversed
                  . indices
                    ( Text.isPrefixOf
                        "farePolicyDriverExtraFeeBounds:"
                        . DAK.toText
                    )
              )

```

make a function (composed lens) and [reuse for all](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).